Makefile

```diff
@@ -123,6 +123,12 @@ build:
 local:
 	@GOOS=darwin common/scripts/gobuild.sh build/_output/bin/$(IMG) ./cmd/manager
 
+openapi:
```

Comment:
hi, can you please try to use `operator-sdk generate openapi` and `operator-sdk generate k8s`  to generate code? which is the approach we are using. 

operator-sdk v0.12.0+

Reply:
generate openapi is gone since v0.13. See release notes here (https://github.com/operator-framework/operator-sdk/releases). I'm using a 0.15 release.

Reply:
I see, let me see if we can move to 0.15 as well. 

At the meantime, can you please change this PR without running the auto-generate, in this way we can merge the other changes first. If I confirm with the team we are moving the operator-sdk version as well, I will just update the auto-gen code at the master branch.

Reply:
Okay. Removing the generated files means that the PR is kinda "incomplete".
But given the situation that migrating to new version of operator-sdk needs some coordination, it is acceptable.

Reply:
great, thanks for the understanding.

Also, I'm seeing your build failed on the Travis, I guess it's due to the fact that forked branch can't get the ENV variable from Travis(a Travis design), such as git_token... due to this, the internal CICD tool failed to be imported...

However, if you branch this repo and I don't believe your have the access to write to it... 

Kind annoy situation over here... 

I will consult with our CICD team for a solution, I believe they open-sourced the CICD tool, if that's the case, you can set up the Travis build on your fork with your own credential.